lib/recurly/invoice.php

```diff
@@ -66,6 +66,16 @@ public function invoiceNumberWithPrefix() {
   }
 
   /**
+   * Enters an offline payment for an invoice
+   * @param Recurly_Transaction additional transaction attributes. The attributes available to set are (payment_method, collected_at, amount_in_cents, description)
+   * @return Recurly_Transaction transaction on success
+   */
+  public function enterOfflinePayment($transaction, $client = null) {
```

Comment:
It's for the tests since this does call a static method. Is there a way around that?